docs/recipes/precompiling-with-webpack.md

```diff
@@ -2,25 +2,33 @@
 
 Translations: [Français](https://github.com/avajs/ava-docs/blob/master/fr_FR/docs/recipes/precompiling-with-webpack.md)
 
-The AVA [readme](https://github.com/avajs/ava#transpiling-imported-modules) mentions precompiling your imported modules as an alternative to runtime compilation, but it doesn't explain how. This recipe shows how to do this using webpack. (This example uses webpack 2.0)
+The AVA [readme](https://github.com/avajs/ava#transpiling-imported-modules) mentions precompiling your imported modules as an alternative to runtime compilation, but it doesn't explain how. This recipe various approaches using webpack. (These examples use webpack 2.0). Multiple approaches are discussed as each has its own pros and cons. You should select the approach that best fits your use case. This might not be necessery once [this](https://github.com/avajs/ava/blob/master/docs/specs/001%20-%20Improving%20language%20support.md) is completed. See the original discussion [here](/avajs/ava/pull/1385).
+
+- [Single test file](#single-test-file)
+- [Multiple test files](#multiple-test-files)
+
+### Single test file
+This is the simplest use case. You might need this if you are [using aliases](https://github.com/avajs/ava/issues/1011).
 
 ###### webpack.config.js
 
 ```js
+const path = require('path');
 const nodeExternals = require('webpack-node-externals');
 
 module.exports = {
-	entry: ['src/tests.js'],
+	entry: ['tests.js'],
```

Comment:
This assumes you only have one test file (named `tests.js`)? Perhaps the file name should be `test.js` then? (At least that's how I name such a file, analogous to having a `test` directory.)